lambdas/thumbnail/src/t4_lambda_thumbnail/__init__.py

```diff
@@ -73,7 +72,7 @@
             'enum': ['pdf', 'pptx']
         },
         'output': {
-            'enum': ['json', 'raw']
+            'enum': ['raw']
         },
```

Comment:
This could be removed once catalog stop send `output` parameter in querystring. Does it make sense to do it in this PR?

Reply:
i think we can remove it here altogether and i'll send a PR removing that from the catalog -- it's trivial